flink_job.go

```diff
@@ -0,0 +1,131 @@
+	CreateFlinkJobRequest struct {
+		JobName   string   `json:"job_name,omitempty"`
+		Statement string   `json:"statement"`
+		Tables    []string `json:"tables"`
+	}
+
+	// CreateFlinkJobResponse Aiven API response
+	// POST https://api.aiven.io/v1/project/<project>/service/<service_name>/flink/job
+	CreateFlinkJobResponse struct {
+		APIResponse
+
+		JobName string `json:"job_name"`
+		JobId   string `json:"job_id"`
+	}
+
+	// DeleteFlinkJobRequest Aiven API request
+	// PATCH https://api.aiven.io/v1/project/<project>/service/<service_name>/flink/proxy/v1/jobs/<job_id>
+	PatchFlinkJobRequest struct {
+		JobId string `json:"job_id"`
+	}
+
+	// GetFlinkJobRequest Aiven API request
+	// GET https://api.aiven.io/v1/project/<project>/service/<service_name>/flink/job/<job_id>
+	GetFlinkJobRequest struct {
+		JobId string `json:"job_id"`
+	}
+
+	// GetFlinkJobRequest Aiven API response
+	// GET https://api.aiven.io/v1/project/<project>/service/<service_name>/flink/proxy/v1/jobs/<job_id>
+	GetFlinkJobResponse struct {
+		APIResponse
+
+		Name           string `json:"name"`
+		JID            string `json:"jid"`
+		IsStoppable    bool   `json:"isStoppable"`
+		Duration       int    `json:"duration"`
+		Now            int    `json:"now"`
+		EndTime        int    `json:"end-time"`
+		StartTime      int    `json:"start-time"`
+		MaxParallelism int    `json:"maxParallelism"`
+		State          string `json:"state"`
+		Plan           struct {
+			JID   string `json:"jid"`
+			Name  string `json:"name"`
+			Nodes []struct {
+				Description         string      `json:"description"`
+				Id                  string      `json:"id"`
+				Operator            string      `json:"operator"`
+				OperatorStrategy    string      `json:"operator_strategy"`
+				OptimizerProperties interface{} `json:"optimizer_properties"`
+				Parallelism         int         `json:"parallelism"`
+			} `json:"nodes"`
+		} `json:"plan"`
+		StatusCounts struct {
+			Canceled     int `json:"CANCELED"`
+			Canceling    int `json:"CANCELING"`
+			Created      int `json:"CREATED"`
+			Deploying    int `json:"DEPLOYING"`
+			Failed       int `json:"FAILED"`
+			Finished     int `json:"FINISHED"`
+			Initializing int `json:"INITIALIZING"`
+			Reconciling  int `json:"RECONCILING"`
+			Running      int `json:"RUNNING"`
+			Scheduled    int `json:"SCHEDULED"`
+		} `json:"status-counts"`
+		Timestamps struct {
+			Canceled     int `json:"CANCELED"`
+			Canceling    int `json:"CANCELING"`
+			Created      int `json:"CREATED"`
+			Deploying    int `json:"DEPLOYING"`
+			Failed       int `json:"FAILED"`
+			Finished     int `json:"FINISHED"`
+			Initializing int `json:"INITIALIZING"`
+			Reconciling  int `json:"RECONCILING"`
+			Running      int `json:"RUNNING"`
+			Scheduled    int `json:"SCHEDULED"`
+		} `json:"timestamps"`
+
+		//TODO: if this is needed, add fields
+		Vertices []interface{} `json:"vertices"`
+	}
+)
+
+// Creates a flink job
```

Comment:
To make go doc happy for all the comments should have a prefix `{function name}`, can you add here //**Create** creates a Flink job?

Reply:
:heavy_check_mark: